cmake/CMakeLists.txt

```diff
@@ -713,6 +713,32 @@ else()
   endif()
 endif()
 
+if(onnxruntime_USE_SVE)
```

Comment:
FYI, this code got moved earlier by another PR:

https://github.com/orlmon01/onnxruntime/blob/6f14c5fc0e3210fd4646e4bbda0eadee3413a470/cmake/CMakeLists.txt#L510-L523

I think this copy can be deleted.